Assets/MRTK/Examples/Demos/HandTracking/Profiles/HandMenuMixedRealityInputSimulationProfile.asset

```diff
@@ -0,0 +1,68 @@
+%YAML 1.1
```

Comment:
What new settings are needed for this profile? Can we re-use the default profiles instead?

Reply:
This is due to us needing the hands to be flat as default to activate the two large menus in the scene.